test/data/validator.output.yaml

```diff
@@ -1,5 +1,8 @@
 jobs:
   main:
+  - annotations:
+      beta.screwdriver.cd/auto_pr_builds: fork-only
+  - description: "This is a description"
```

Comment:
I'm kinda confused. Shouldn't this be in the same level as settings and environments?